README.md

```diff
@@ -143,6 +133,29 @@ model.print_topic_distribution(
 
 </center>
 
+#### Automated Topic Naming
+
+Turftopic now allows you to automatically assign human readable names to topics using LLMs or n-gram retrieval!
+
+```python
+from turftopic import KeyNMF
+from turftopic.namers import OpenAITopicNamer
+
+model = KeyNMF(10).fit(corpus)
```

Comment:
```suggestion
model = KeyNMF(n_topics=10).fit(corpus)
```

Reply:
Same as before, n_topics is not an existing argument. I copied scikit-learn's API in every way here.